tests/Traits/KeyPairTestingTrait.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ *  This program is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU General Public License
+ *  as published by the Free Software Foundation; under version 2
+ *  of the License (non-upgradable).
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ *  Copyright (c) 2020 (original work) Open Assessment Technologies S.A.
+ */
+
+declare(strict_types=1);
+
+namespace OAT\SimpleRoster\Tests\Traits;
+
+/**
+ * KeyPairTestingTrait
+ *
+ * @package OAT\SimpleRoster\Tests\Traits
+ */
+trait KeyPairTestingTrait
```

Comment:
I know that we discussed about this yesterday, but I believe we can remove this trait. Let's pregenerate the keys in config/secrets/test` folder and commit them into the repository.

Reply:
Done